tests/unit/test_synapse_api.py

```diff
@@ -102,12 +102,45 @@ def test_get_nonce_success(mock_requests):
 
 def test_get_nonce_error(monkeypatch: pytest.MonkeyPatch):
     """
-    arrange: set User parameters.
-    act: register the user.
+    arrange: none.
```

Comment:
Similar to above

Reply:
And I suppose applicable for the others as well

Reply:
Arrange changed.